docs/deployment/README.md

```diff
@@ -266,40 +228,72 @@ dev:                            # deployment name
 
-### Set up an environment file
+**Deploy `iam` stack**[^1]
 
-_If you're adding a new deployment to an existing configuration repository or re-deploying an existing Cumulus configuration you should skip to [Deploy the Cumulus Stack](deployment-readme#deploy), as these values should already be configured._
+```bash
+  $ DEPLOYMENT=<iam-deployment-name> \
+      AWS_REGION=<region> \ # e.g. us-east-1
+      AWS_PROFILE=<profile> \
+      npm run deploy-iam
+```
 
-Copy `app/.env.sample` to `app/.env` and add CMR/earthdata client [credentials](deployment-readme#credentials):
+**Note**: If this deployment fails check the deployment details in the AWS Cloud Formation Console for information. Permissions may need to be updated by your AWS administrator.
 
-```shell
-  CMR_USERNAME=cmrusername
-  CMR_PASSWORD=cmrpassword
-  EARTHDATA_CLIENT_ID=clientid
-  EARTHDATA_CLIENT_PASSWORD=clientpassword
-  VPC_ID=someid
-  VPC_CIDR_IP=0.0.0.0/0
-  AWS_SUBNET=somesubnet
-  AWS_ACCOUNT_ID=0000000
-  AWS_REGION=awsregion
-  TOKEN_SECRET=tokensecret
+If the `iam` deployment command  succeeds, you should see 7 new roles in the [IAM Console](https://console.aws.amazon.com/iam/home):
+
+* `<prefix>-ecs`
+* `<prefix>-lambda-api-gateway`
+* `<prefix>-lambda-processing`
+* `<prefix>-scaling-role`
+* `<prefix>-steprole`
+* `<prefix>-distribution-api-lambda`
+* `<prefix>-migration-processing`
+
+The same information can be obtained from the AWS CLI command: `aws iam list-roles`.
+
+The `iam` deployment also creates an instance profile named `<stack-name>-ecs` that can be viewed from the AWS CLI command: `aws iam list-instance-profiles`.
+
+--------------
+
+## Deploy the Database stack
+
+This section will cover deploying the DynamoDB and ElasticSearch resources.
+Reminder: ElasticSearch is optional and can be disabled using `es: null` in your `config.yml`.
+
+**Deploy `db` stack**
+
+```bash
+  $ DEPLOYMENT=<cumulus-deployment-name> \
+      AWS_REGION=<region> \ # e.g. us-east-1
+      AWS_PROFILE=<profile> \
+      npm run deploy-db
 ```
 
-The `TOKEN_SECRET` is a string value used for signing and verifying [JSON Web Tokens (JWTs)](https://jwt.io/) issued by the API. For security purposes, it is strongly recommended that this be a 32-character string.
+--------------
 
-Note that the `.env.sample` file may be hidden, so if you do not see it, show hidden files.
+## Deploy the Cumulus stack
 
-For security it is highly recommended that you prevent `apps/.env` from being accidentally committed to the repository by keeping it in the `.gitignore` file at the root of this repository.
+This section will cover deploying the primary Cumulus stack, containing compute resources, workflows and all other AWS resources not covered in the two stacks above.
```

Comment:
I think it's worth noting somewhere at the end of this "deploy iam -> db -> app manually" that you've also set up a `deploy-all` command that will do them all at once.

Reply:
👍 6c267ec